examples/computer_vision/cifar10_pytorch_inference/model_def.py

```diff
@@ -84,6 +93,8 @@ def __init__(self, context: PyTorchTrialContext) -> None:
 
         # TODO: Load your trained model. Below are example approaches.
 
+        # XXX: this is a bad idea; users frequently try to load checkpoints via checkpoint export
+        #      but due to DET-3448 that is highly likely to result in opaque, difficult errors.
```

Comment:
We could remove this, but I'm worried that folks are going to be trying to load their model programmatically anyway. Should we keep the code but add a user-facing warning, similar to the comment here?

Do we have an alternative way of loading in a checkpoint that's not problematic that we could suggest?

Reply:
Yes, setting `searcher.source_checkpoint_uuid` should work fine for checkpoints in the system.